package.json

```diff
@@ -21,6 +21,7 @@
     "refresh": "node scripts/lerna clean --scope \"@theia/*\" --yes && yarn run bootstrap",
     "clean": "node scripts/lerna run clean --scope \"@theia/*\" --parallel",
     "build": "node scripts/lerna run build --scope \"@theia/*\" --stream",
+    "docs": "rimraf docs/api && lerna run docs",
```

Comment:
that rimraf  docs/api is not used anymore it should be that either we output to docs/api or we rimraf docs
I would rather have it that we output to docs/api

Reply:
so it should be `docs": "lerna run docs`?